product_docs/docs/pgd/5/deploy-config/deploy-manual/deploying/08-using-pgd-cli.mdx

```diff
@@ -124,129 +123,143 @@ Once PGD CLI is configured, you can use it to get PGD-level views of the cluster
 
 ### Check the health of the cluster
 
-The [`check-health`](../../../cli/command_ref/pgd_check-health) command provides a quick way to view the health of the cluster:
+The [`cluster show --health`](../../../cli/command_ref/cluster/show/#options) command provides a quick way to view the health of the cluster:
 
 ```
-pgd check-health
+pgd cluster show --health
 __OUTPUT__
-Check      Status Message
------      ------ -------
-ClockSkew  Ok     All BDR node pairs have clockskew within permissible limit
-Connection Ok     All BDR nodes are accessible
-Raft       Ok     Raft Consensus is working correctly
-Replslots  Ok     All BDR replication slots are working correctly
-Version    Ok     All nodes are running same BDR versions
-```
+Connections
```

Comment:
This command is a tabular output now by default. Please update the output or add `-o markdown` flag if the report format is desired